### ISSUE TITLE
do not include *.ts files inside npm

### ISSUE THREAD

**Issue by TrejGun** [Kresten/winston-papertrail-transport#7]
my tsc tries to transpile them even if i have node_modules exluded and can't find types for both glossy and triple-beam

**Comment by Kresten** [comment]
I have excluded ts files in 1.0.8, let me know if that fixed it :)

**Comment by TrejGun** [comment]
Thanks for a fast response
yes it did the trick
but now another problem
package.json says 
```
  "types": "index.d.ts",
```
while file is in dist dir
```
  "types": "./dist/index.d.ts",
```
and you probably want to exclude tests too
```
  "files": [
    "dist/*",
    "!dist/test"
  ],
```

**Comment by Kresten** [comment]
I was a little too fast, copy/pasted from one of my other packages, sorry!
I have published 1.0.9 with a better package.json!

**Comment by TrejGun** [comment]
thanks! this works like magic
